lrlex/src/lib/ctbuilder.rs

```diff
@@ -414,7 +414,7 @@ where
             LexerKind::LRNonStreamingLexer => (
                 "LRNonStreamingLexerDef",
                 format!(
-                    "LRNonStreamingLexerDef<{storaget}, {lexertypest}<{storaget}>>",
+                    "LRNonStreamingLexerDef<{storaget}, {lexertypest}>",
```

Comment:
Good catch, seems like an obscure one!

Reply:
I had misread this as *adding* the `<{storaget}>` rather than *removing* it and figuring that it was previously always generating code using the default. Just so I understand, this is removing `<{storaget}>`, because `type_name::<LexerTypesT>()` actually already contains the `<{storaget}>` part.  So this works now for both the cases where a non-default, and default `StorageT` are provided. The following playground link seems to indicate so.

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=47ef81f885c6b78264af6bc88bc3fc42

Reply:
Correct: before this fix we ended up producing things like `LexerTypes<u8><u8>`.